ql/src/test/org/apache/hadoop/hive/ql/exec/spark/TestSparkInvalidFileFormat.java

```diff
@@ -30,11 +30,13 @@
 import org.apache.hadoop.hive.ql.session.SessionState;
 
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 
 import java.io.File;
 import java.io.IOException;
 
+@Ignore("HIVE-22944: Kryo 5 upgrade conflicts with Spark, which is not supported anymore")
```

Comment:
Does this mean this is a breaking change for Spark?

Reply:
yes, it is
maybe I could hack with that, but it would be painful: using kryo5 for tez execution and kryo4 for spark execution